docs/pages/setup/helm-reference/teleport-kube-agent.mdx

```diff
@@ -326,6 +368,53 @@ You can specify multiple databases by adding additional list elements.
   You can see a list of all the supported [values which can be used in a Teleport database service configuration here](../../database-access/reference/configuration.mdx).
 </Admonition>
 
+## `dbResources`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `list` | `[]` | When `db` chart role is used at least one of `databases`, `awsDatabases`, `dbResources` is required. |
+
+`dbResources` is a YAML list object detailing the resource selectors of the databases that should be proxied by Teleport Database access.
+
+You can specify multiple selectors by adding additional list elements.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  dbResources:
+  - labels:
+      "env": "prod"
+      "engine": "postgres"
+  - labels:
+      "env": "test"
+      "engine": "mysql"
+  ```
+
+  <Admonition type="note" title="YAML formatting">
+    YAML is very sensitive to correct spacing. When specifying lists in a `values.yaml` file, you might like
+    to [use a linter](https://codebeautify.org/yaml-validator) to validate your YAML list and ensure that it is correctly formatted.
+  </Admonition>
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $
+  --set "dbResources[0].labels.env=prod" \
+  --set "dbResources[0].labels.engine=postgres" \
+  --set "dbResources[1].labels.env=test" \
+  --set "dbResources[0].labels.engine=mysql"
+  ```
+
+  <Admonition type="note">
+    Note that when using `--set` syntax, YAML list elements must be indexed starting at `0`.
+  </Admonition>
+
+  </TabItem>
+</Tabs>
+
+<Admonition type="tip" title="Example">
```

Comment:
Same comment as above re: this admonition

Reply:
I tried something